lib/Doctrine/ORM/Persisters/Entity/AbstractEntityInheritancePersister.php

```diff
@@ -82,12 +82,11 @@ protected function getSelectColumnSQL($field, ClassMetadata $class, $alias = 'r'
     /**
      * @param string $tableAlias
      * @param string $joinColumnName
-     * @param string $className
      * @param string $type
      *
      * @return string
      */
-    protected function getSelectJoinColumnSQL($tableAlias, $joinColumnName, $className, $type)
+    protected function getSelectJoinColumnSQL($tableAlias, $joinColumnName, $type)
```

Comment:
This is a minor BC break, and needs to be documented in [`UPGRADE.md`](https://github.com/doctrine/doctrine2/blob/master/UPGRADE.md)